performance/theme_runner.rb

```diff
@@ -70,4 +68,55 @@ def compile_and_render(template, layout, assigns, page_template, template_file)
       content_for_layout
     end
   end
+
+  # the following 3 methods are for when we want to benchmark just the 'render'
+
+  # compile_tests should be called before the benchmark is run, and benchmark_render
+  # should be called to to benchmark just the render
+
+  def compile_tests
+    # Dup assigns because will make some changes to them
+    assigns = Database.tables.dup
+
+    @compiled_tests = []
+    @tests.each do |liquid, layout, template_name|
+      # Compute page_tempalte outside of profiler run, uninteresting to profiler
+      page_template = File.basename(template_name, File.extname(template_name))
+      @compiled_tests << compile_and_save(liquid, layout, assigns, page_template, template_name)
+    end
+  end
+
+  def compile_and_save(template, layout, assigns, page_template, template_file)
```

Comment:
a version of compile that didn't render after compiling but instead saves the compilation results would be useful, so that's why I have `compile_and_save`. If we were to remove `compile_and_render`, that means changing the `run` benchmark to call `compile_all_tests` followed by `render` - would that be ok?